core/types/block.go

```diff
@@ -239,11 +244,21 @@ type extblockV1 struct {
 	IncomingReceipts CXReceiptsProofs
 }
 
+// includes staking transaction
+type extblockV2 struct {
+	Header           *block.Header
+	Txs              []*Transaction
+	Stks             []*staking.StakingTransaction
+	Uncles           []*block.Header
```

Comment:
can you remove Uncles. we don't need it.

Reply:
removed for extV3